README.md

```diff
@@ -64,6 +64,26 @@ Capybara.javascript_driver = :selenium_billy # Uses Firefox
 `:webkit_billy` for headless specs when using puffing-billy for other local rack apps.
 See [this phantomjs issue](https://github.com/ariya/phantomjs/issues/11342) for any updates.
 
+#### Advanced setup
+If you need to configure additional settings on the driver -- for example, to
+simulate browsing at different screen sizes -- you can manually register a driver
+and pass it an options object.
```

Comment:
This a little misleading since only 2/4 of the drivers want an Options object.  Let's be more clear here and then I'd list out all four examples in sub-sections.